.ci.yaml

```diff
@@ -1597,7 +1597,7 @@ targets:
 
   - name: Linux_android complex_layout_android__compile
     recipe: devicelab/devicelab_drone
-    presubmit: false
+    presubmit: true
```

Comment:
Test is passing on presubmit. This LGTM once you remove the presubmit run here.

Reply:
Done